src/families/stellar/libcore-buildOperation.js

```diff
@@ -1,16 +1,25 @@
 // @flow
 
 import type { CoreOperation } from "../../libcore/types";
+import { libcoreBigIntToBigNumber } from "../../libcore/buildBigNumber";
 
 async function stellarBuildOperation({
   coreOperation
 }: {
   coreOperation: CoreOperation
 }) {
   const stellarLikeOperation = await coreOperation.asStellarLikeOperation();
-  const stellarLikeTransaction = await stellarLikeOperation.getRecord();
-  const hash = await stellarLikeTransaction.getTransactionHash();
-  return { hash };
+  const stellarLikeRecord = await stellarLikeOperation.getRecord();
+  const stellarLikeTransaction = await stellarLikeOperation.getTransaction();
+  const hash = await stellarLikeRecord.getTransactionHash();
+  const transactionSequenceNumberRaw = await stellarLikeTransaction.getSourceAccountSequence();
```

Comment:
Do you a different value for each operation ? 
I'm not sure if this one return the correct Sequence number or the Account Sequence.
If it's not we probably need to ask some dev to libcore to get the right sequence number

Reply:
checked with @pollastri-pierre , getSourceAccountSequence correctly returns the transaction ledger and not the account's. phew.